CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ BUG FIXES:
 * Fix large number will be truncated in plan ([#1382](https://github.com/opentofu/opentofu/pull/1382))
 * S3 backend no longer requires to have permissions to use the default 'env:' workspace prefix ([#1445](https://github.com/opentofu/opentofu/pull/1445))
 * Fixed a crash when using a conditional with Twingate resource ([1446](https://github.com/opentofu/opentofu/pull/1446))
+* Fix for `tofu init` failure when test have spaces in their name. ([1489](https://github.com/opentofu/opentofu/pull/1489))
```

Comment:
I think we should make it clear in the changelog here (before merging) that we no longer accept spaces in test run names.

Let's not just say it's a "fix", let's explain what the fix is.

Reply:
Done. Thanks!